src/com/projectkorra/rpg/RPGMethods.java

```diff
@@ -42,14 +44,14 @@ public static boolean checkEveryInterval(World world) {
 	}
 	
 	public static void cycleAvatar(BendingPlayer bPlayer) {
-		if (Bukkit.getOnlinePlayers().size() <= 1) return; //Don't bother with 1 person on...
```

Comment:
Yes? But the comment that was on it was wrong, I have not changed the code of it.
It said don't bother with 0 players online, but the actually comment should be don't both with 1 or less persons online, like the code is saying.